src/theme/palette.js

```diff
@@ -12,24 +12,27 @@ export const pinColor = {
 };
 
 export const designColor = {
-  white: '#FFFFFF',
+  white: grayscale[0],
```

Comment:
Getting this error: 

`ReferenceError: Cannot access 'grayscale' before initialization`